docs/MatrixOne/Reference/SQL-Reference/Explain/explain.md

```diff
@@ -13,7 +13,7 @@ where option can be one of:
     FORMAT { TEXT | JSON }
 ```
 
-For 0.5.1, only `TEXT` format is supported.
+For 0.6.0, only `TEXT` format is supported.
```

Comment:
Please verify if these limitations are still the same. @domingozhang

Reply:
json is not supported